python/src/deltachat/testplugin.py

```diff
@@ -519,6 +519,7 @@ def prepare_account_from_liveconfig(self, configdict) -> Account:
         ac = self.get_unconfigured_account()
         assert "addr" in configdict and "mail_pw" in configdict, configdict
         configdict.setdefault("bcc_self", False)
+        configdict.setdefault("configured_mvbox_folder", "DeltaChat")
```

Comment:
Maybe the same needs to be done for the sentbox

Reply:
The reason is that `configure_folders()` sets `configured_mvbox_folder` to "DeltaChat" only in two cases:
- If `mvbox_move` is set (but it's not set initially in the test). Then "DeltaChat" is created.
- If "DeltaChat" already exists (also not the case).

I still think, for this particular test we need to create the "DeltaChat" folder manually.

Reply:
Done. Set `mvbox_move` initially so that "DeltaChat" is created, then disable it

Reply:
You're right, that was another one footgun. I added a fix for it. Now the test is more simple as mvbox is auto-created. I checked, it still reproduces the original problem (if IO is not auto-restarted). But maybe another one test is needed for the second commit.